packages/mobile/src/components/FeeDrawer.tsx

```diff
@@ -20,6 +20,8 @@ interface Props {
   isExchange?: boolean
   securityFee?: BigNumber
   exchangeFee?: BigNumber
+  dekFee?: BigNumber
+  showDekfee?: boolean
```

Comment:
nit: I'm always wondering about the appropriate wiring convention for acronyms in pascalCase, looks like this one follows what JavaScript does in general (`XmlHttpRequest`, etc).
But we also have `deriveDEK` later on.
Really not important, just dumping my thoughts.

Reply:
Yeah that's a good point, I guess I've been inconsistent! I'll rename deriveDEK to deriveDek